hera_pspec/uvpspec.py

```diff
@@ -335,6 +335,29 @@ def get_kparas(self, spw, little_h=True):
 
         return k_para
 
+    def get_blpairs(self):
+        """
+        Returns a list of all blpair tuples in the data_array.
+        """
+        return [self.blpair_to_antnums(blp) for blp in np.unique(self.blpair_array)]
+
+    def get_all_keys(self):
+        """
+        Returns a list of all possible tuple keys in the data_array.
```

Comment:
It would be good to say what the keys are, i.e. (spectral window, baseline pair, polarisation)